### ISSUE TITLE
Context Menu unintentionally disabled by default

### ISSUE THREAD

**Issue by aigyptios** [namespace-ee/react-calendar-timeline#769]
In Item.js, the context menu handler has an if statement checking for whether there is a handler passed in as a prop:
https://github.com/namespace-ee/react-calendar-timeline/blob/9182d7eb9a1cef7027dae26c4ceb485ed7cf4b07/src/lib/items/Item.js#L499-L505

However, the condition is superfluous because a handler is always passed in to the `<Items>` component, starting in Timeline.js:
https://github.com/namespace-ee/react-calendar-timeline/blob/9182d7eb9a1cef7027dae26c4ceb485ed7cf4b07/src/lib/Timeline.js#L824
https://github.com/namespace-ee/react-calendar-timeline/blob/9182d7eb9a1cef7027dae26c4ceb485ed7cf4b07/src/lib/Timeline.js#L606-L611

Because it will never be undefined, context menu click event handlers will be passed into the `<div>` but will be `defaultPrevented` automatically, meaning that we aren't able to open the default browser context menu in any case.

Would it be possible to set the `onItemContextMenu` prop for the `<Items>` component conditionally, only if an actual handler is defined? Such as the following:

```
<Items
  ...
  onItemContextMenu={ this.props.onItemContextMenu ? this.contextMenuClickItem : undefined }
  ... 
  />
```

**Comment by Ilaiwi** [comment]
@aigyptios I have refactored this in beta I think. Could you check the `develop` branch?

**Comment by aigyptios** [comment]
Hi @Ilaiwi, I'm not seeing the changes you're referring to on the `develop` branch. Could you send me a link? Thank you!

**Comment by dsgipe** [comment]
@Ilaiwi is this still being maintained? If not, can I be added as a maintainer so we can keep updating various pieces of it.

**Comment by dsgipe** [comment]
@Ilaiwi ping

**Comment by Ilaiwi** [comment]
released in 0.28.0
